frontend/src/service-worker.js

```diff
@@ -48,25 +50,20 @@ registerRoute(
   createHandlerBoundToURL(process.env.PUBLIC_URL + '/index.html'),
 );
 
-if (process.env.NODE_ENV === "production" || process.env.NODE_NEV === "dev-pwa") {
+if (process.env.NODE_ENV === 'production') {
   registerRoute(
     ({url}) => {
       const urlString = url.toString();
-      if (urlString.startsWith('https://cdn.jsdelivr.net/')) {
-        return true;
-      } else if (urlString.startsWith('https://cdnjs.cloudflare.com')) {
-        return true;
-      } else if (urlString.startsWith('https://stackpath.bootstrapcdn.com/bootstrap/')) {
-        // TODO: move bootstrap inline rather than getting from CDN
-        return true;
-      } else if (urlString.startsWith('https://futurecoder.io')) {
-        return true;
-      } else if (urlString.startsWith('https://fonts.googleapis.com')) {
-        return true;
-      } else if (urlString.startsWith('https://pyodide-cdn2.iodide.io')) {
-        return true;
+      if (
+        urlString.startsWith('https://cdn.jsdelivr.net/')
+        || urlString.startsWith('https://cdnjs.cloudflare.com')
+        || urlString.startsWith('https://pyodide-cdn2.iodide.io')
```

Comment:
Could you add comments for when these are used? What happened to fonts.googleapis.com?

Reply:
@alexmojaki I realized that one was only used in `homepage` not within `frontend`, so it seems unnecessary. But I can put it back.

Adding comments 👍